packages/graphql-generator/src/__tests__/models.test.ts

```diff
@@ -1,45 +1,70 @@
-import { generateModels, GenerateModelsOptions, ModelsTarget } from '..';
+import { generateModels, generateModelsSync, GenerateModelsOptions, ModelsTarget } from '..';
 import { readSchema } from './utils';
 
-describe('generateModels', () => {
-  describe('targets', () => {
-    const targets: ModelsTarget[] = ['java', 'swift', 'javascript', 'typescript', 'dart', 'introspection'];
-    targets.forEach(target => {
-      test(`basic ${target}`, async () => {
+const generators = ['generateModels', 'generateModelsSync'] as const;
+
+async function runGenerator(generator: (typeof generators)[number], options: GenerateModelsOptions) {
```

Comment:
Rather than have two copies of the tests, here we use a string to run th same tests for both the sync and async versions.